zaza/openstack/utilities/upgrade_utils.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Collection of functions to support upgrade testing."""
+import re
+import logging
+import collections
+import zaza.model
+
+
+SERVICE_GROUPS = collections.OrderedDict([
+    ('Core Identity', ['keystone']),
+    ('Control Plane', [
+        'aodh', 'barbican', 'ceilometer', 'ceph-mon', 'ceph-fs',
+        'ceph-radosgw', 'cinder', 'designate',
+        'designate-bind', 'glance', 'gnocchi', 'heat', 'manila',
+        'manila-generic', 'neutron-api', 'neutron-gateway', 'placement',
+        'nova-cloud-controller', 'openstack-dashboard']),
+    ('Data Plane', [
+        'nova-compute', 'ceph-osd', 'swift-proxy', 'swift-storage'])
+])
+
+UPGRADE_EXCLUDE_LIST = ['rabbitmq-server', 'percona-cluster']
```

Comment:
This is really an exclude from openstack upgrade list isn't it ? So, maybe rename

Reply:
Yeah - this is an exclude list for openstack upgrades, will rename it :)